src/GitHub.VisualStudio/Services/UsageService.cs

```diff
@@ -32,7 +32,7 @@ public bool IsSameDay(DateTimeOffset lastUpdated)
 
         public bool IsSameWeek(DateTimeOffset lastUpdated)
         {
-            return GetIso8601WeekOfYear(lastUpdated) == GetIso8601WeekOfYear(DateTimeOffset.Now);
+            return GetIso8601WeekOfYear(lastUpdated) == GetIso8601WeekOfYear(DateTimeOffset.Now) && lastUpdated.Year != DateTimeOffset.Now.Year;
```

Comment:
Is this right? The method is `IsSameWeek` not `IsDifferentWeek`.

Reply:
We're using it as different week everywhere (i.e., is it the same week? no? then send week data). Of course if you run VS on the same week of a different year, it's going to think it's the same week...

Reply:
But with this change, `IsSameWeek(DateTimeOffset.Now) == false` - that's not correct surely? Today is the same week as today.

Reply:
Oh, you mean the wrong comparison in the year because fat fingers? Funny how there's no tests that caught me doing that backwards...